src/commands/misc/qrcode/read.js

```diff
@@ -0,0 +1,27 @@
+const { Command, CommandError, SwitchbladeEmbed } = require('../../../')
+const fetch = require('node-fetch')
+
+module.exports = class QRCodeRead extends Command {
+  constructor (client) {
+    super(client, {
+      name: 'read',
+      aliases: ['r'],
+      parentCommand: 'qrcode',
+      parameters: [{
+        type: 'string', full: true, missingError: 'commands:qrcode.subcommands.read.noImage'
+      }]
+    })
+  }
+
+  async run ({ t, channel, author, message }, text) {
+    const body = await fetch(`http://api.qrserver.com/v1/read-qr-code/?fileurl=${message.attachments.first() ? message.attachments.first().url : text}`).then(res => res.json())
```

Comment:
image parameter!

Reply:
Yes for god's sake use an image parameter